jupyterhub/templates/proxy/netpol.yaml

```diff
@@ -18,38 +18,22 @@ spec:
     - Egress
   ingress:
     - ports:
-      - protocol: TCP
-        port: 80
-      - protocol: TCP
-        port: 443
-      {{- if not $autoHTTPS }}
-      - protocol: TCP
-        port: 8000
-      {{- end }}
+      - port: http
       {{- if or $manualHTTPS $manualHTTPSwithsecret}}
-      - protocol: TCP
-        port: 8443
+      - port: https
       {{- end }}
-      # kube-lego /healthz
-      - protocol: TCP
-        port: 8080
-      # nginx /healthz
-      - protocol: TCP
-        port: 10254
```

Comment:
This is to allow ingress to the deploy/proxy's pod, but that pod doesn't expose these ports anyhow. This is not relevant any more.